mutacc/utils/vcf_handler.py

```diff
@@ -4,6 +4,21 @@
 
 import yaml
 
+from .constants import (
+    SEMICOLON,
```

Comment:
This is a quality of life thing, but if you order the imports alphabetically - you can just scan to the import you are looking for or is missing (and find duplicates) without having to read the entire block or until you happen on the one that you want.

Reply:
This applies to all code that do not requiring a specific order